packages/client/components/TeamPromptMeeting.tsx

```diff
@@ -22,23 +86,80 @@ const TeamPromptMeeting = (props: Props) => {
     graphql`
       fragment TeamPromptMeeting_meeting on TeamPromptMeeting {
         ...TeamPromptTopBar_meeting
+        id
+        ...useMeeting_meeting
+        phases {
+          id
+          phaseType
+          stages {
+            ... on TeamPromptResponseStage {
+              id
+              teamMember {
+                id
+                preferredName
+                picture
+              }
+            }
+          }
+        }
       }
     `,
     meetingRef
   )
 
+  const maybeTabletPlus = useBreakpoint(Breakpoint.FUZZY_TABLET)
+  const teamMembers = useMemo(() => {
+    return meeting.phases
+      .flatMap((phase) => phase.stages)
+      .flatMap((stage) => stage.teamMember)
+      .map((teamMember) => {
+        if (!teamMember) {
+          return null
+        }
+
+        return {
+          ...teamMember,
+          key: teamMember.id
+        }
+      })
+      .filter(isNotNull)
+  }, [meeting.phases])
+  const transitioningTeamMembers = useTransition(teamMembers)
+  const {safeRoute} = useMeeting(meeting)
+  if (!safeRoute) return null
+
   return (
     <MeetingStyles>
       <MeetingArea>
         <Suspense fallback={''}>
           <MeetingContent>
             <MeetingHeaderAndPhase hideBottomBar={true}>
               <TeamPromptTopBar meetingRef={meeting} />
-              <PhaseWrapper>
-                <Link title='My Dashboard' to='/meetings'>
-                  <img alt='Parabol' src={logoMarkPurple} />
-                </Link>
-              </PhaseWrapper>
+              <Prompt>What are you working on today? Stuck on anything?</Prompt>
+              <ErrorBoundary>
+                <ResponsesGridContainer maybeTabletPlus={maybeTabletPlus}>
+                  <ResponsesGrid>
+                    {transitioningTeamMembers.map((teamMember) => {
+                      const {child, onTransitionEnd, status} = teamMember
+                      const {id, picture, preferredName} = child
+
+                      return (
+                        <TeamMemberResponse
+                          key={id}
```

Comment:
+1 Would it make sense to use `child.key` here?  That would guarantee consistency between the `key` used by `useTransition` and the `key` used by React.